Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -10074,7 +10074,7 @@ a_Player:OpenWindow(Window);
 							Type = "boolean",
 						},
 					},
-					Notes = "If a map with the specified ID exists, calls the CallbackFunction for that map. The CallbackFunction has the following signature: <pre class=\"prettyprint lang-lua\">function Callback({{cMap|Map}})</pre> Returns true if the map was found and the callback called, false if map not found.",
+					Notes = "If a map with the specified ID exists, calls the CallbackFunction for that map. The CallbackFunction has the following signature: <pre class=\"prettyprint lang-lua\">function Callback({{cMap|Map}})</pre> Returns true if the map was found and the callback called, false if map not found. Note: If an error occurs in the callback, the callback returns true.",
```

Comment:
Do you mean the callback function or the API function? The API function returns false, if the callback returns true.
https://api.cuberite.org/cWorld.html#ForEachEntity_1